include/pmacc/particles/memory/dataTypes/Particle.hpp

```diff
@@ -94,10 +94,10 @@ namespace pmacc
          * ATTENTION: The pointer must be the last member to avoid local memory usage
          *            https://github.com/ComputationalRadiationPhysics/picongpu/pull/762
          */
```

Comment:
As we are already touching this part of the code. Is this still a thing? If so, I think we're violating it here. If not, remove.